README.md

```diff
@@ -100,6 +48,47 @@ mutant --include lib --require virtus --use rspec Virtus::Attribute.build
 mutant --include lib --require virtus --use rspec Virtus::Attribute#type
 ```
 
+Configuration
+-------------
+
+Occasionally mutant will produce a mutation with an infinite runtime.  When this happens
```

Comment:
I'd prefer we go for one space after `.`. For the reasons:

* Its more usual in documentation I read these days
* I'd never remind myself to use two spaces for consistency.